posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,808 @@
+
+Versionless features simplify your server configuration by providing are a generic way to specify Open Liberty features without having to specify a feature version. These new features are available for the MicroProfile, Jakarata EE, and Java EE platforms. When you use versionless features, you must specify a corresponding versioned platform element in your `server.xml` file.
+
+For example, instead of specifying `servlet-6.1` in your `server.xml` file and having to figure out which other feature versions are compatible with Servlet 6.1, you can specify a platform version and `servlet`. The platform that you specify resolves all versionless features to a compatible version.
+
+The following `server.xml` file configuration uses versionless features with `MicroProfile-5.0` specified as the platform element.
+
+[source,xml]
+----
+    <!-- Enable features -->
+    <featureManager>
+        <platform>microProfile-5.0</platform>
+        <feature>mpHealth</feature>
+        <feature>mpMetrics</feature>
+    </featureManager>
+----
+
+Another example is specifying a Java EE platform of `javaee-8.0` with associated versionless features that are defined for `servlet`, `jpa`, and `jaxrs`:
+
+[source,xml]
+----
+    <!-- Enable features -->
+    <featureManager>
+        <platform>javaee-8.0</platform>
+        <feature>servlet</feature>
+        <feature>jpa</feature>
+        <feature>jaxrs</feature>
+    </featureManager>
+----
+
+Check out the full collection of available platforms and versionless features in the <<platform-ref, lists at the end of this post>>. Stay tuned for more information versionless features in a forthcoming blog post and full documentation when versionless features are released for general availability (GA).
+
+=== Known limitations
+
+* Error handling scenarios return generic messages.
+* Feature installation scenarios that use  command-line tools, including Maven and Gradle plug-ins, are not supported.
+
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/29122
+// Contact/Reviewer: benjamin-confino
+// // // // // // // //
+
+[#cookie]
+== Keep using third party cookies with CHIPS
+
+To increase privacy and reduce tracking, link:https://developers.google.com/privacy-sandbox/3pcd/[Google Chrome announced it would phase out third-party cookies in 2025]. However, some sites that are designed with third-party cookies in mind are broken by this change. Chrome provides link:https://developers.google.com/privacy-sandbox/3pcd/prepare/test-for-breakage[documentation] to help you test whether your sites are affected. If you are, a handful of options are provided to mitigate this change. One such option is called CHIPS: Cookies Having Independent Partitioned State.

```

Comment:
```suggestion
To increase privacy and reduce tracking, link:https://developers.google.com/privacy-sandbox/3pcd/[Google Chrome announced it would phase out third-party cookies in 2025]. Although, as of July 22, 2024, link:https://privacysandbox.com/news/privacy-sandbox-update/[Chrome stated they may scrap the phase out plan] due to regulatory concerns.  Users instead would be given the option to block third party cookies via their browser.  Partitioned would be needed for these use cases. However, some sites that are designed with third-party cookies in mind are broken by this change. Chrome provides link:https://developers.google.com/privacy-sandbox/3pcd/prepare/test-for-breakage[documentation] to help you test whether your sites are affected. If you are, a handful of options are provided to mitigate this change. One such option is called CHIPS: Cookies Having Independent Partitioned State.

```